tests/devices/bf/device_bf_test.py

```
@@ -2,10 +2,38 @@
+from midealocal.exceptions import SocketException
+from midealocal.message import MessageType
+
+
+def _build_device() -> MideaBFDevice:
+    """Build a Midea BF device."""
+    return MideaBFDevice(
+        name="Test BF Device",
+        device_id=1,
+        ip_address="192.168.1.1",
+        port=12345,
+        token="AA",
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "token"

(S106)